src/OpenIddict/OpenIddictExtensions.cs

```diff
@@ -38,6 +38,7 @@ public static IdentityBuilder AddOpenIddict<TApplication>([NotNull] this Identit
                 // and the MVC modules are enabled.
                 builder.UseAssets();
                 builder.UseMvc();
+                builder.UseSecurityHeaders();
```

Comment:
OK. I tend to add to the end by default because I don't always know what setup needs done before my code.